controllers/authController.js

```diff
@@ -128,21 +129,28 @@ export const forgotPasswordController = async (req, res) => {
   try {
     const { email, answer, newPassword } = req.body;
     if (!email) {
-      res.status(400).send({ message: "Emai is required" });
+      res.status(400).send({ message: "Email is required" });
     }
     if (!answer) {
-      res.status(400).send({ message: "answer is required" });
+      res.status(400).send({ message: "Answer is required" });
     }
     if (!newPassword) {
       res.status(400).send({ message: "New Password is required" });
     }
     //check
-    const user = await userModel.findOne({ email, answer });
+    const user = await userModel.findOne({ email });
     //validation
     if (!user) {
       return res.status(404).send({
         success: false,
-        message: "Wrong Email Or Answer",
+        message: "Email is not registered",
+      });
+    }
+    // Validate answer separately
+    if (user.answer !== answer) {
+      return res.status(400).send({
+        success: false,
+        message: "Incorrect security answer",
```

Comment:
I think we can retain the same message in this case? Wrong Email Or Answer to also prevent them from guessing the security answer